driver/worker.go

```diff
@@ -0,0 +1,103 @@
+package driver
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// worker executes a unit of work and has a one-to-one relationship with a client
+// that will be responsible for executing the work. Currently worker is not safe for
+// concurrent use by multiple goroutines
```

Comment:
I think you can remove the stuff about client from the comment since this PR removes the client from worker

Reply:
👍 thanks for drawing my attention to the godocs. I made a pass and updated a number of other ones that were missing or affected by the refactor.